buildSrc/src/main/java/com/google/firebase/gradle/plugins/DackkaPlugin.kt

```diff
@@ -67,7 +156,7 @@ abstract class DackkaPlugin : Plugin<Project> {
         )
     }
 
-    // TODO(b/243534168): Refactor when fixed, so we no longer need stubs
+    // TODO(b/243954517): Refactor when fixed, so we no longer need stubs
```

Comment:
I think this bug just got fixed this morning, but afaicr there was another one related to hiding the hidden superclass, do you recall if we filed it?

Reply:
I think this is the right one https://issuetracker.google.com/issues/243324828

Reply:
Yeah, the initial bug was [b/243324828](https://issuetracker.google.com/issues/243324828).
Another one was the one you filed (which you're probably thinking of), which is this: [b/243954517](https://b.corp.google.com/issues/243954517).

Both those bugs need to be fixed for us to remove stubs, so we still need the original bug to fixed. It seems like that might be sooner rather than later, shout out to the androidx team. Let me change the comment to reflect that.